### ISSUE TITLE
Docs: Create Jekyll Collection for Meshery Adapters

### ISSUE THREAD

**Issue by leecalcote** [layer5io/meshery#470]
**Current State:**
Meshery Adapters are listed on a few different pages. Their multiple listings become fragile as new adapters are created and existing adapters move between alpha, beta, stable, and continue to version (e.g. v2.0.0) from there.

**Desired State:**
Ideally, this information is gathered into a _data [collection](https://jekyllrb.com/docs/collections/) as a single source of truth to be referenced across these various pages.

- [ ] Create a new collection for adapters.
- [ ] Supplant static adapter lists with a collection iterator.

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

**Comment by leecalcote** [comment]
@subhamkrai would you like to give this one a shot?

**Comment by schillingjerrid** [comment]
I'll take a swing at this.

**Comment by schillingjerrid** [comment]
Hey @leecalcote, sorry it has been taking a while on this issue. I ran into a few setbacks but I got past them today.

I have a question about integrating the collection into one of the pages where adapters were listed. On the main page we list available adapters, then in-progress adapters, and finally help-wanted adapters. I've turned these into a table tentatively with a status variable for each collection item, but wanted your feedback before I create the PR.

New adapter list on home page:
<img width="1124" alt="Screen Shot 2019-11-19 at 10 17 32 PM" src="https://user-images.githubusercontent.com/18037669/69211283-cfb4a880-0b1a-11ea-944b-cd5af19c8e65.png">

Current adapter list on home page:
<img width="1131" alt="Screen Shot 2019-11-19 at 10 17 47 PM" src="https://user-images.githubusercontent.com/18037669/69211301-dc390100-0b1a-11ea-99c1-64377130cbfc.png">
